app/connector/jira/pom.xml

```diff
@@ -175,6 +166,19 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <configuration>
+                    <!-- jira-rest-java-client-core is a hard transitive dependency on jira-rest-java-client-api
+                        should be at compile scope, but maven-dependency-plugin will throw a WARN when validating it -->
+                    <usedDependencies>
+                        <dependency>com.atlassian.jira:jira-rest-java-client-core</dependency>
```

Comment:
if `com.atlassian.jira:jira-rest-java-client-core` is declared at `runtime` scope this would not be needed.

Reply:
runtime scope works when integration starts, but fails when open JiraVerifierExtension in IDE as it can't resolve AsynchronousJiraRestClientFactory from jira-rest-java-cliet-core